uyuni-docs-helper

```diff
@@ -66,6 +66,13 @@ print_incorrect_syntax() {
   exit 1
 }
 
+get_dir_abs_path() {
+  (
```

Comment:
Are these brackets necessary?

Reply:
Yes, they are needed so this runs in a subshell. That way we can change the directory without the need of going back to the original one.

Reply:
Thank you for the explanation.